library/src/test/java/com/getstream/sdk/chat/EmptyTextMessageTest.java

```diff
@@ -0,0 +1,35 @@
+package com.getstream.sdk.chat;
+
+import com.getstream.sdk.chat.utils.StringUtility;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class EmptyTextMessageTest {
```

Comment:
null case is still not covered. It is crucial thing because right now it might trigger NPE and crash of an app.